Jenkinsfile

```diff
@@ -342,6 +355,37 @@ pipeline {
             }
 
         }
+        stage('Math functions expressions test') {
```

Comment:
I would move this in the "Cmdstan tests" parallel stage and rename that to CmdStan & Math tests. Other than that looks good, assuming tests pass.